ocfweb/stats/accounts.py

```diff
@@ -32,7 +46,7 @@ def _get_account_stats():
 
     # Some accounts don't have creation times, so we assume that as UIDs
     # increase, so does time. We use this assumption to fill in the gaps.
-    start_date = date(1989, 1, 1)
+    start_date = date(1995, 8, 21)
```

Comment:
what's the reasoning behind changing this date?

Reply:
The data was all zeroes before that time so I changed it to make the graph look better visually.